src/Item/Item.model.ts

```diff
@@ -1,5 +1,5 @@
 import { Model, SQL, raw } from 'decentraland-server'
-import { Collection } from '../Collection'
+import { Collection } from '../Collection/Collection.model'
```

Comment:
<img width="353" alt="image" src="https://user-images.githubusercontent.com/692077/158733374-834b3b9b-6840-4eff-88c4-b79410c6fbf8.png">

circular dep?

Reply:
Sadly there are a lot of circular dependencies between item and collection. I can't remember if this one was impacting our tests, but I tried to reduce them as much as possible.